modules/web-console/frontend/app/components/ignite-chart/components/chart-no-data/controller.ts

```diff
@@ -31,18 +34,33 @@ export default class IgniteChartNoDataCtrl implements ng.IOnChanges, ng.IOnDestr
 
     handleClusterInactive: boolean;
 
+    connectionState$ = this.AgentManager.connectionSbj.pipe(
+        pluck('state'),
+        distinctUntilChanged(),
+        tap((state) => {
+            if (state === 'AGENT_DISCONNECTED')
+                this.destroyChart();
+        })
+    );
+
     cluster$ = this.AgentManager.connectionSbj.pipe(
         pluck('cluster'),
+        distinctUntilChanged(),
         tap((cluster) => {
             if (_.isNil(cluster) && !this.AgentManager.isDemoMode()) {
                 this.destroyChart();
                 return;
             }
 
-            if (cluster.active === false && this.handleClusterInactive)
+            if (!_.isNil(cluster) && cluster.active === false && this.handleClusterInactive)
```

Comment:
Why import lodash if you can simply `!!cluster && cluster.active === false` or even `cluster && cluster.active === false`.

Reply:
@Klaster1  Fixed this and other similar cases.

Reply:
Don't forget to remove lodash imports, if no longer needed.

Reply:
Sure, Removed unused imports.